tests/conftest.py

```diff
@@ -162,7 +162,7 @@ def get_tbinfo(request):
         raise ValueError("testbed and testbed_file are required!")
 
     testbedinfo = cache.read(tbname, 'tbinfo')
-    if not testbedinfo:
+    if testbedinfo is cache.NOTEXIST:
```

Comment:
There are other similar checks that need to be updated.

Reply:
I double-checked if there are no modules that use `FactsCache` left unchanged, could you help confirm this?

Reply:
I just confirmed all covered. Please ignore.